software/controller/lib/actuators/pwm_actuator.h

```diff
@@ -22,14 +22,12 @@ limitations under the License.
 
 class PwmActuator : public Actuator {
  public:
-  PwmActuator(const PwmPin pin, const Frequency pwm_freq, const char *name, const char *help,
-              float cal_0 = 0.0f, float cal_1 = 1.0f)
-      : Actuator(name, help, cal_0, cal_1), pwm_(pin, pwm_freq){};
+  PwmActuator(GPIO::PwmChannel channel, Frequency pwm_freq, Frequency cpu_frequency,
+              const char *name, const char *help, float cal_0 = 0.0f, float cal_1 = 1.0f)
+      : Actuator(name, help, cal_0, cal_1), pwm_pin_(channel, pwm_freq, cpu_frequency){};
 
-  void initialize_pwm(Frequency cpu_frequency) { pwm_.initialize(cpu_frequency); }
-
-  void set(const float value) { pwm_.set(get_value(value)); }
+  void set(const float value) { pwm_pin_.set(get_value(value)); }
```

Comment:
again let's make explicit that `get_value` is from superclass

Reply:
okay